packages/assets-controllers/src/NftController.ts

```diff
@@ -107,6 +121,7 @@ export interface NftMetadata {
   description: string | null;
   image: string | null;
   standard: string | null;
+  tokenUri?: string;
```

Comment:
This change is not required for this feature but I'm thinking it might be a useful thing to add.